### PR TITLE
Adding config entry for disconnect message, if mojang is down

### LINKED ISSUE #809: Change "Authentication servers are down" message
Paper has a few helpful message configurations where Spigot/Bukkit fail to cover (e.g. The kick message when a player is "flying"). This request is the same, but tied to the "unable to connect" message when Mojang's Authentication Servers are down.

**Current default**
"Authentication servers are down. Please try again later, sorry!"
_(this makes players contact the server and tell them their stuff is broken...when in reality it's out of their control and a more descriptive message would prevent that frustration)_

**I would personally make it**
"Mojang's Authentication Servers are currently down. Check here for status: http://xpaw.ru/mcstatus/"

### PR DESCRIPTION
**Pull request by phenomax**
This PR resolves #809.

I'll just test it on my local system and remove the WIP then.

### CODE REVIEW

File: Spigot-Server-Patches/0234-Adding-config-entry-for-disconnect-message-if-mojang.patch [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+--- a/src/main/java/com/destroystokyo/paper/PaperConfig.java
++++ b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+@@ -270,4 +270,9 @@ public class PaperConfig {
+     private static void suggestPlayersWhenNull() {
+         suggestPlayersWhenNullTabCompletions = getBoolean("settings.suggest-player-names-when-null-tab-completions", suggestPlayersWhenNullTabCompletions);
+     }
++
++    public static String authenticationServersDownMessage = "Authentication servers are down. Please try again later, sorry!";
++    private static void authenticationServersDownMessage() {
++        flyingKickPlayerMessage = getString("messages.authenticationServersDown", authenticationServersDownMessage);
```
Comment (AlfieC) [inline Spigot-Server-Patches/0234-Adding-config-entry-for-disconnect-message-if-mojang.patch]: flyingKickPlayerMessage?

**Review by AlfieC — CHANGES_REQUESTED** [review]

File: Spigot-Server-Patches/0234-Adding-config-entry-for-disconnect-message-if-mojang.patch (hunk 1 above)
Comment (phenomax) [reply Spigot-Server-Patches/0234-Adding-config-entry-for-disconnect-message-if-mojang.patch]: Copy paste detected :D

**Comment by kashike** [comment]
I'm going to handle this differently.

**Comment by jamierocks** [comment]
couldn't you just use a custom resource pack to do this?

Seems like a bad idea to break things for non-English speakers.

**Comment by kashike** [comment]
https://github.com/PaperMC/Paper/commit/d43da9fc7083e49f628f9a040708d4341534ccea

**Pull request closed without merging**